src/event.ts

```diff
@@ -85,6 +85,10 @@ interface ErrorBase<T extends PackageReference> extends EventBase<T> {
   type: 'failure';
 }
 
+interface ScriptlessError {
```

Comment:
Isn't there always at least a package json path? Why can't `PackageJsonPathError` and `InvalidConfigSyntax` have `extends ErrorBase<PackageReference>>`?